### ISSUE TITLE
New apple orchards

### ISSUE THREAD

**Issue by Rakso69** [Sh0wdown/UnofficialCrusaderPatch#75]
Forget what I've been speaking of before about apple orchards in the farm section. :) Here is a new idea of mine: Multi-seasonal apple orchards! So that out of 8 trees: 2 are dead-green, 2 are white, 2 are green and 2 are red! So that every food gatherer can work for himself in an endless cycle, and doesn't need to wait for the trees to grow!

- Here is an exaple from Stronghold 1 Mission 18, but I'd suggest the method to be as above:
8 trees: 2 dead-green, 2 white, 2 green and 2 red.
![seasons](https://user-images.githubusercontent.com/45049934/48901839-faf8af00-ee56-11e8-93a8-ce05193a0d58.jpg)

**Comment by PitchNeeded** [comment]
I think apple production is fine as it is, the production is decent enough if you have a decent number of apple farms (I mean just watch how many apples The Snake manages to gather) it is not supposed to be as efficient as bread production after all!

**Comment by Rakso69** [comment]
@PitchNeeded Not really. It isn't fine. Starvation from apple orchards come in very fast if you play for an enormous economy making them not worth at all(If You don't do the season trick). I think this could make both the AI more efficient, aswell would leave a decision for the human player, if would he prefer a bread economy or stay with apple orchards. Without it everyone does see only bread. beacuse meat is limited, cheese sucks and is more efficent to make gold from leather armour instead of feed peasants with it. Apple are useless without the trick as soon You hit 150 population or so, you're already getting less apple than it's being consumed and starvation starts soon(on normal rations!). Concluding it as bread being the only effective source of raise your food rations.
Also on the other side myself I consider it unfair that multiplayer players are aware of the season pass and newcommers or singleplayer players usually are not. The chances should be balanced, and running on apple till like +/- 350 population takes less effort to estabilish food compared to bread eco, and let's You focus on other things.
Naturally bread is way better than apple, but I think that definately there should be always a kind of choose of what you want to use, not only one top strategy. I'd suggest buff cheese farms even, that would be interesting! :P

**Comment by ByBurton** [comment]
You should NOT balance the game around massive industry, with 80 + population.
The apple farms are more than enough, I would even say they are a little too strong.

**Comment by Mizo5119** [comment]
Honestly I'd just make the Apple farms slightly smaller. Their efficiency is fine, but just not viable on most maps.

Especially if you're like me and like to mix food for more popularity, it becomes tedious.

**Comment by Rakso69** [comment]
@ByBurton Why not? This game is not only singleplayer and AI, but also multiplayer. It's worth considering every inch and aspect, not playing on economy with less than 80 peasants, beacuse AI does that. lmao

**Comment by Rakso69** [comment]
@Mizo5119 I think you should explain your idea better.
I doubt You mean to say to make the apple farms smaller like (hut + 3 trees only), that would be too broken imo. :D 

But one thing is right, orchards area should be fixed @Sh0wdown . It's so strange that apple orchards are bigger while build, but after placing them one line of bottom and right is deleted, so more apple are able to be build if You start building them from top-left corner. :/ It should be equal from all sides, just as in SH1 is.

**Comment by Rakso69** [comment]
@Mizo5119 Cool idea. I like that. Go ahead.

- But note that: if 2 workers, then imo one should bring 3 apple to the granary instead of 4, as they do vanilla. Beacuse if 2 orchards workers, then they would bring 8 together, and that's the equal ammount of bread brought by a baker. So my suggestion would be 6 apple from one farm then.

Mixing food is already pretty OP strategy, as different food kinds slow down the consumption of the other ones, so the granary is able to hold in for really long and amass food greatly.

**Comment by ByBurton** [comment]
@Mizo5119 Apple farms are bigger, because that balances out their effectiveness.
If you compare them to cheese farms: those have less production, but are smaller.

**Comment by ByBurton** [comment]
@Rakso69 But the focus should be on single player, as a lot more people play that.

**Comment by Rakso69** [comment]
@ByBurton lmao. Wrong! The focus should be over about everything.

**Comment by PitchNeeded** [comment]
I think we should take both multiplayer and single player into consideration. I suppose the proposed smaller apple farms might be good for the AI too, it would allow the apple growing AI (of which there are a lot) to place more farms close to their castle rather than having to place some of their apple farms near enemy castles.

**Comment by Rakso69** [comment]
@ByBurton He should look for multiplayer even more than singleplayer. Balancing multiplayer benefits singleplayer. Simple as that. Always key-tip when it comes to modding.

**Comment by ByBurton** [comment]
This is for Nere to decide.

**Comment by Sh0wdown** [comment]
I could look after adjusting the footprint, as the farms need more space when creating them, than they take afterwards. Multi-seasonal trees are a bit too complex a think.
